src/Docfx.Common/Json/System.Text.Json/SystemTextJsonUtility.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+#nullable enable
+
+namespace Docfx.Common;
+
+/// <summary>
+/// Utility class for JSON serialization/deserialization.
+/// </summary>
+public static class SystemTextJsonUtility
+{
+    /// <summary>
+    /// Default JsonSerializerOptions options.
+    /// </summary>
+    public static readonly JsonSerializerOptions DefaultSerializerOptions;
+
+    /// <summary>
+    ///  Default JsonSerializerOptions options with indent setting.
+    /// </summary>
+    public static readonly JsonSerializerOptions IndentedSerializerOptions;
+
+    static SystemTextJsonUtility()
+    {
+        DefaultSerializerOptions = new JsonSerializerOptions()
+        {
+            // DefaultBufferSize = 1024 * 16, // TODO: Set appropriate buffer size based on benchmark.(Default: 16KB)
+            AllowTrailingCommas = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull | JsonIgnoreCondition.WhenWritingDefault,
```

Comment:
Feels a bit safer to only use `JsonIgnoreCondition.WhenWritingNull`. `JsonIgnoreCondition.WhenWritingDefault` can ignore default enum values which may not be desirable.

Reply:
This setting intended to align options used by following code.
(Because `JsonSerializerOptions` instance is recommended to be shared when same option used)
https://github.com/dotnet/docfx/blob/b7285fcdfb04e6f4132d702217588de6236bfa4c/src/Docfx.Build/ApiPage/ApiPage.cs#L149

As you noted.
It's seems better to use only `WhenWritingNull`. I've removed `WhenWritingDefault` (https://github.com/dotnet/docfx/pull/9832/commits/194506188c8d0b86a9d54c09173bced59fb04853)